okta-hosted-login/README.md

```diff
@@ -32,11 +32,16 @@ Plug these values into the `mvn` commands used to start the application.
 cd okta-hosted-login
 mvn -Dokta.oauth2.issuer=https://{yourOktaDomain}/oauth2/default \
     -Dokta.oauth2.clientId={clientId} \
-    -Dokta.oauth2.clientSecret={clientSecret} \
-    -Dokta.oauth2.postLogoutRedirectUri={absoluteLogoutRedirectUri} # (optional) configure this property to enable SSO logout. 
```

Comment:
Why is this `postLogoutRedirectUri` property removed? I know its optional but still good to note it I think.

Reply:
I saw it's hard coded in the code plus it is not mentioned in the other readme. The readme at our java spring sdk does mention this so I thought we can simplify the cli here in sample. If we need to know detail, the instinct would go to underlining SDK page.

```
// set logout URL
.and().logout().logoutSuccessUrl("/")
```